webclient/http2/src/main/java/io/helidon/webclient/http2/Http2ClientStream.java

```diff
@@ -302,6 +302,7 @@ private Http2FrameData readOne() {
                     Http2Headers http2Headers = Http2Headers.create(this,
                                                                     connection.getInboundDynamicTable(),
                                                                     requestHuffman,
+                                                                    null,
```

Comment:
This should not be done, as the API is public.. So either create two methods (one with existing headers, one without them), or do it differently

Reply:
Noted, fixed